istio/cmd/app/cmd.go

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app
```

Comment:
move istio/cmd to /cmd  rename app to istio

Reply:
discussed with @little-cui , we agreed to put all the stuff but docs in the istio folder to decouple with service center.

Reply:
the reason is？

Reply:
this is a tool for service center and istio integration, the user could choose to use it or not, and this tool runs in a standalone mode, therefore it is not necessary to put main function in the `/cmd` folder